lib/flutter_stories.dart

```diff
@@ -150,6 +159,7 @@ class _StoryState extends State<Story> with SingleTickerProviderStateMixin {
   AnimationController _controller;
   int _currentIdx;
   bool _isInFullscreenMode = false;
+  bool displayCloseIcon = false;
```

Comment:
It's not a good decision to store it as a property instead of simply checking for `onClose != null` at line 306, because if you pass the `onClose` callback sometime later after the initial build, it won't be propagated to the widget tree and close button will not be visible.